Sources/StreamChatSwiftUI/ChatChannel/MessageList/MessageView.swift

```diff
@@ -128,6 +129,7 @@ public struct MessageTextView<Factory: ViewFactory>: View {
         trailingPadding: CGFloat = 16,
         topPadding: CGFloat = 8,
         bottomPadding: CGFloat = 8,
+        lineSpacing: CGFloat = 1,
```

Comment:
I couldn't find any official docs about the default value of `lineSpacing`. Not sure if it's 1, and if that applies for every dynamic font size. Therefore, there's a risk that this might be a breaking change.